python/cuml/preprocessing/LabelEncoder.py

```diff
@@ -34,6 +34,10 @@ def _enforce_npint32(y: cudf.Series) -> cudf.Series:
     return y
 
 
+def _get_nvstring_from_series(y: cudf.Series):
+    return y._data[list(y._data.keys())[0]]._nvstrings
```

Comment:
@dantegd you can grab it a bit less awkwardly if you want by using the public `nvstrings` instead with `y._column.nvstrings`

Reply:
awesome! The more I look at the code in LabelEncore, I think I'll encapsulate even more things so thinking of submitting it in a small follow up